cmd/updateIntegrationArtifactConfiguration_test.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/SAP/jenkins-library/pkg/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+type updateIntegrationArtifactConfigurationMockUtils struct {
+	*mock.ExecMockRunner
+	*mock.FilesMock
+}
+
+func newUpdateIntegrationArtifactConfigurationTestsUtils() updateIntegrationArtifactConfigurationMockUtils {
+	utils := updateIntegrationArtifactConfigurationMockUtils{
+		ExecMockRunner: &mock.ExecMockRunner{},
+		FilesMock:      &mock.FilesMock{},
+	}
+	return utils
+}
+
+func TestRunUpdateIntegrationArtifactConfiguration(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Successfull update of Integration Flow configuration parameter test", func(t *testing.T) {
+		config := updateIntegrationArtifactConfigurationOptions{
+			Host:                   "https://demo",
+			OAuthTokenProviderURL:  "https://demo/oauth/token",
+			Username:               "demouser",
+			Password:               "******",
+			IntegrationFlowID:      "flow1",
+			IntegrationFlowVersion: "1.0.1",
+			ParameterKey:           "myheader",
+			ParameterValue:         "def",
+		}
+
+		httpClient := httpMockCpis{CPIFunction: "UpdateIntegrationArtifactConfiguration", ResponseBody: ``, TestType: "Positive"}
+
+		err := runUpdateIntegrationArtifactConfiguration(&config, nil, &httpClient)
+		// assert
+		assert.NoError(t, err)
```

Comment:
I agree with @marcusholl

Reply:
I would like to suggest again t check also the url and the method of the http request.

Reply:
@mayurmohan Would you like to comment here why this is not necessary?

Reply:
Imagine somebody makes some changes to your code. The http traffic should stay as it is, but unfortunately the author had a bad day and e.g. the url is changed by error. When we check for the expected url we can see that at the earlierst possible point in time: during the unit tests. Otherwise we will see it later during the integration tests, hopefully in case the are not omitted. The next point in time where we can see it afterwards is a customer message complaining about some not working functionallity ...